examples/dashboard/dashboard.py

```diff
@@ -28,7 +31,6 @@
 from webargs import fields
```

Comment:
How should we handle the dashboard's dependence on webargs?  Should we just state that in the README?  I don't think we'd necessarily want to make webargs a dependency of datashader, at least not until conda supports optional dependencies.